recipe/scripts/activate.bat

```diff
@@ -1,4 +1,4 @@
 if not defined GDAL_DATA (
-  set GDAL_DATA="%~dp0\..\..\..\Library\share\gdal"
+  set "GDAL_DATA=%~dp0\..\..\..\Library\share\gdal"
```

Comment:
Note that setting of environment variables from activate.bat may still be broken on Windows by default - see https://github.com/conda/conda-env/issues/218 and linked PR. If you copy the changes over manually (https://github.com/conda/conda-env/pull/219) it seems to work fine.